schemas/mod-source-record-manager/deleteJobExecutionsRespDto.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Response status of soft delete JobExecution entities",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "job_execution_log_details": {
```

Comment:
Why properties here named with using _ convention when other properties in repository named with camel case?

Reply:
@SatishkumarPurswani This will assist: https://dev.folio.org/guidelines/naming-conventions/